classes/widgets/class-suggested-tasks.php

```diff
@@ -145,11 +145,12 @@ public function enqueue_scripts() {
 
 		// Check if current date is between Feb 12-16.
 		$confetti_options = [];
-		$current_date     = \gmdate( 'Y-m-d' );
+		$year             = '2025';
```

Comment:
Could we just make this `date('Y')` ? means we don't have to do it again next year :)

Reply:
Haha I said *shouldn't* :)